grouper/fe/handlers/group_join.py

```diff
@@ -161,11 +169,12 @@ def _get_member(self, member_choice):
             resource = Group
 
         if resource is None:
-            return
+            return None
 
         return self.session.query(resource).filter_by(name=member_name, enabled=True).one()
 
     def _get_choices(self, group):
+        # type: (Group) -> List[Tuple[str, ...]]
```

Comment:
Yeah, I'll add a comment about that. They are `Tuple[str, str]`, but mypy doesn't understand that because of the `* 2` construction and thinks they're `Tuple[str, ...]` instead.